ansible/ci-build-collector.yml

```diff
@@ -13,6 +13,8 @@
       community.general.make:
         chdir: "{{ lookup('env', 'GITHUB_WORKSPACE') }}"
         target: image
+      # ensure this action is printed
+      tags: [print_action]
```

Comment:
thinking about it, we'll also need a task to always print the output of this make target (or conditionally on failure only?)

Reply:
I think printing the output on failure is what we want, success just means the image was built correctly.

Reply:
it might be worth doing that with the tests as well